src/DeepNgToDo/Tests/Frontend/angular/services/DeepNgDoToService.spec.js

```diff
@@ -67,18 +190,67 @@ describe('Services', function() {
     expect(deepNgToDoService.completedCount).toBe(0);
   });
 
-  it('Check deleteTodo() method returns promise', function() {
+  it('Check deleteTodo() method returns promise with reponse.isError', function() {
     let todo = {
       Title: 'first todo',
       Completed: false,
     };
-    deepNgToDoService.todoList = [todo,];
-    expect(deepNgToDoService.todoNumber).toBe(1);
-    expect(deepNgToDoService.deleteTodo(todo)).not.toEqual(null);
+    let error = null;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).